src/api/call-api.test.ts

```diff
@@ -1,291 +1,291 @@
-import * as abi from '@api3/airnode-abi';
-import * as adapter from '@api3/airnode-adapter';
-import * as node from '@api3/airnode-node';
-import * as ois from '@api3/airnode-ois';
-import { ethers } from 'ethers';
-import { callApi } from './call-api';
+// import * as abi from '@api3/airnode-abi';
```

Comment:
@acenolaza  I've temporarily commented out these tests as `callApi` changed so much but I wanted to open this PR before I lose track of all the changes... sorry for it all being in one commit but I wasn't able to make it build otherwise :pray: 

I'll fix these tests next

Reply:
No worries. I assumed that was the case. 👍🏻